config/database.yml.example

```diff
@@ -10,6 +10,6 @@ development:
 # Do not set this db to the same as development or production.
 test:
   adapter: postgresql
-  database: arcdata-test
+  database: travis_ci_test
```

Comment:
Instead of change the database name here, why not change the one in .travis.yml to `arcdata-test`?

Reply:
oops, yeah. Of course. Sorry